core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -693,7 +694,6 @@ class GroupCoordinator(val brokerId: Int,
   }
 
   private def onMemberFailure(group: GroupMetadata, member: MemberMetadata) {
```

Comment:
Can we rename this method as well? Maybe just `removeMemberFromGroup` or something like that?